frontend/components/layout/Header.vue

```diff
@@ -42,6 +42,7 @@
                             :has-accepted-terms="hasAcceptedTerms"
                             @update:isModalOpen="$emit('update:isModalOpen', $event)"
                             @changeWalletType="$emit('changeWalletType', $event)"
+                            @openWalletModal="$emit('openWalletModal')"
```

Comment:
I still feel like we need to rework some of these modal names. We currently have `walletModal` for managing DAI and `walletSelectModal` for selecting a wallet, as far as I am aware.

Maybe renaming `walletModal` to `manageDaiModal` would make sense? Or a different name.

Reply:
I think `changeWalletType` sounds reasonable, as well as `openWalletModal` since it's called `WalletModal`, right?